Userland/Libraries/LibJS/Heap/Heap.cpp

```diff
@@ -332,7 +332,7 @@ void Heap::sweep_dead_cells(bool print_report, Core::ElapsedTimer const& measure
 
         dbgln("Garbage collection report");
         dbgln("=============================================");
-        dbgln("     Time spent: {} ms", time_spent.to_milliseconds());
+        dbgln("     Duration spent: {} ms", time_spent.to_milliseconds());
```

Comment:
This looks like it was lined up with the other rows before, now it isn't.

Although, one could argue that this should stay as "Time", as it's clear that "time spent" refers to a duration and since that's the usual way of saying that. The alternative would be to just call it "Duration" and remove the "spent".

Reply:
This should have accidentally been find-replaced.